### PR TITLE
Docs: New index page with list of all visualizations and panels 

### PR DESCRIPTION
**Pull request by torkelo**
* [x] New index page with an index list of all visualizations grouped into 4 categories 
* [x] Examples section with some new screenshots and links to play site demos 
* [x] Some new better screenshots 
* [x] Change side menu of this section to "Visualisations"  (Was "Visualization options") 

Website images PR 
https://github.com/grafana/website/pull/5225

localhost_3002_docs_grafana_next_panels_visualizations_ (1)

### CODE REVIEW

File: docs/sources/panels/visualizations/pie-chart-panel.md [hunk 1]
```diff
@@ -6,9 +6,9 @@ weight = 850
 
 # Pie chart
 
-The pie chart displays reduced series, or values in a series, from one or more queries, as they relate to each other, in the form of slices of a pie. The arc length, area and central angle of a slice are all proportional to the slices value, as it relates to the sum of all values. This type of chart is best used when you want a quick comparison of a small set of values in an aesthetically pleasing form.
+{{< figure src="/static/img/docs/pie-chart-panel/pie-chart-example.png" max-width="1200px" lightbox="true" caption="Table visualization" >}}
```
Comment (pkolyvas) [inline docs/sources/panels/visualizations/pie-chart-panel.md]: Should `caption="Pie chart vizualization"`? Also, if small things like this are ok for me to fix directly I'm happy to do so.

Edit, made the fix.

**Review by pkolyvas — APPROVED** [review]
Made some minor tweaks - I think it's a very valuable addition, and one customers have been asking for. 

It's probably worth revisiting next week with the docs team, as we incorporate some screenshots of updated and new visualizations, but let's merge this!

**Comment by torkelo** [comment]
@pkolyvas Thanks 

> Made some minor tweaks - I think it's a very valuable addition, and one customers have been asking for.
> It's probably worth revisiting next week with the docs team, as we incorporate some screenshots of updated and new visualizations, but let's merge this!

Yea I think could need some continued iterations, but it's a good start :)

**Comment by torkelo** [comment]
@pkolyvas Added docs for the new color mode support, had some merge issues so might have lost 1-2 of your edits on the new graph color scheme fikle 

localhost_3002_docs_grafana_next_panels_visualizations_time-series_graph-color-scheme_

File: docs/sources/panels/visualizations/_index.md [hunk 2]
```diff
@@ -1,10 +1,75 @@
 +++
-title = "Visualization options"
+title = "Visualizations"
 weight = 420
 +++
 
-# Visualization options
+# Visualizations
```
Comment (ryantxu) [inline docs/sources/panels/visualizations/_index.md:6]: maybe?
```suggestion
# Visualization panels
```

Do we want to make it more clear that "panels" == "visualizations"?  And adding new "panel plugins" adds more visualization options?

File: docs/sources/panels/visualizations/_index.md [hunk 3]
```diff
@@ -1,10 +1,75 @@
...
 
-Grafana offers a variety of visualizations to suit different use cases. This section of the documentation lists the different visualizations available in Grafana and their unique options.
+Grafana offers a variety of different visualizations to suit different use cases. This section of the documentation lists the different visualizations available in Grafana, their unique options and use cases. You can add more panel types with [plugins]({{< relref "../../plugins/_index.md" >}}).
```
Comment (ryantxu) [inline docs/sources/panels/visualizations/_index.md:8]:
```suggestion
Grafana offers a variety of visualizations to support different use cases. This section of the documentation highlights the built-in panels, their options and typical usage. You can add more visualization types by installing panel [plugins]({{< relref "../../plugins/_index.md" >}}) from the plugin catalog.
```

**Review by ryantxu — APPROVED** [review]
This looks great -- maybe a few tweaks to make it more clear that "visualizations" are "panel plugins"

**Comment by torkelo** [comment]
@ryantxu @pkolyvas added your suggestions 

created a "Get more" heading under the list with the call out install plugins 

Screenshot from 2021-07-14 17-39-02

**Pull request merged**